src/arch/dotproductsse.cpp

```diff
@@ -32,14 +32,65 @@ namespace tesseract {
 // Uses Intel SSE intrinsics to access the SIMD instruction set.
 #if defined(FAST_FLOAT)
 TFloat DotProductSSE(const TFloat *u, const TFloat *v, int n) {
-  TFloat total = 0.0;
-  for (int k = 0; k < n; ++k) {
-    total += u[k] * v[k];
-  }
-  return total;
+	int max_offset = n - 4;
```

Comment:
Please use spaces for indentation as in the rest of the code.

Reply:
Apologies. WILCO. 👍